### DISCUSSION TITLE
Frozen python module

### DISCUSSION THREAD

**Discussion by gswifort** [CEXT-Dan/PyRx#51] [Ideas]
Have you considered the possibility of loading a python module/package compiled into .exe into Autocad (e.g. using `pyinstaller`)?

**Comment by CEXT-Dan** [comment]
I’ve seen PyInstaller in passing, I skipped over it because I didn’t think it would work. 
In general, you can’t load a .exe into a .exe because they both have their own process.
The performance would be exactly back to PyAutoCAD .ugh, unless I’m missing something 

I’ve seen PyImport_ImportFrozenModule and PyImport_ImportFrozenModuleObject in the API
There’s not a lot of documentation on this, how would this help us?

**Reply by CEXT-Dan** [reply]
I have thought about the need to distribute packages. Maybe have some sort of source protection 
Nothing beyond that though. Is that something that should be done in this module? Or through other means such as PyPI

**Reply by gswifort** [reply]
I think `PyImport_ImportFrozenModule` is for importing compiled `.dll` libraries, not `.exe`
I also thought it wouldn't work, but I thought you might have an idea :)
